bs/src/tablecloth.mli

```diff
@@ -1,68 +1,476 @@
+(** Documentation for tablecloth.mli *)
```

Comment:
Don't need this line anymore.

Reply:
According to http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sec351, “The first special comment of the file is the comment associated  with the whole module.“ Without this comment, the first comment for the `<|` operator would become the module comment.

Reply:
Ah, gotcha!